### PR TITLE
feat: add common uuid and logger functions

### PR DESCRIPTION
**Pull request by ServiceDog**
Also:
- added an example log4rs yml file for testing purposes
- added back in the docker-compose files
- updated references to Arrow-air in all docs

### CODE REVIEW

File: docs/conops.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
```
Comment (owlot) [inline docs/conops.md:1]: should point to https://github.com/aetheric-oss/.github/blob/main/assets/doc-banner.png

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
-![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
+![Aetheric Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
```
Comment (owlot) [inline README.md:1]: should point to https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png

File: lib/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
-![Arrow Banner](https://github.com/Arrow-air/.github/raw/main/profile/assets/arrow_v2_twitter-banner_neu.png)
+![Aetheric Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
```
Comment (owlot) [inline lib/README.md:1]: should point to https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png

File: README.md (hunk 2 above)
Comment (ServiceDog) [reply README.md]: fixed

**Review by owlot — APPROVED** [review]

**Pull request merged**